<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testeQtModernGLClass</class>
 <widget class="QMainWindow" name="testeQtModernGLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testeQtModernGL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="GLcanvas" name="glDisplay" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>401</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Selecione ponto1 e ponto final</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="logPoints">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>200</height>
        </size>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string>Ponto id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>x</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>z</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="logEdges">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>200</height>
        </size>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>80</number>
       </attribute>
       <column>
        <property name="text">
         <string>Aresta id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>P0</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>P1</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="logFaces">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>105</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>200</height>
        </size>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string>Face id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>A0</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>A1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>A2</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>10</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionFit2World"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionConvexHull"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionPoint"/>
   <addaction name="actionEdge"/>
   <addaction name="actionFace"/>
   <addaction name="actionBoundaryCondition"/>
   <addaction name="actionSolver"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionFit2World">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fit2World</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionPoint">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Point</string>
   </property>
  </action>
  <action name="actionEdge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edge</string>
   </property>
  </action>
  <action name="actionFace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Face</string>
   </property>
  </action>
  <action name="actionBoundaryCondition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>boundaryCondition</string>
   </property>
  </action>
  <action name="actionSolver">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>actionSolver</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionConvexHull">
   <property name="text">
    <string>convexHull</string>
   </property>
   <property name="toolTip">
    <string>crie o fecho convexo </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLcanvas</class>
   <extends>QWidget</extends>
   <header location="global">glcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="testeqtmoderngl.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>logPoints</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>mainToolBar</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
